staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -212,21 +218,29 @@ func (o *DebugOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []st
 // Validate checks that the provided debug options are specified.
 func (o *DebugOptions) Validate(cmd *cobra.Command) error {
 	// CopyTo
-	// These flags are exclusive to --copy-to
-	if len(o.CopyTo) == 0 {
+	if len(o.CopyTo) > 0 {
+		if len(o.Image) == 0 && len(o.SetImages) == 0 && len(o.Args) == 0 {
+			return fmt.Errorf("Must specify --image, --set-image or command arguments.")
+		}
+		if len(o.Args) > 0 && len(o.Container) == 0 && len(o.Image) == 0 {
+			return fmt.Errorf("Must specify an existing container or new image when specifying args.")
+		}
+	} else {
+		// These flags are exclusive to --copy-to
 		switch {
 		case o.Replace:
 			return fmt.Errorf("--replace may only be used with --copy-to.")
 		case o.SameNode:
 			return fmt.Errorf("--same-node may only be used with --copy-to.")
+		case len(o.SetImages) > 0:
+			return fmt.Errorf("--set-image may only be used with --copy-to.")
+		case len(o.Image) == 0:
+			return fmt.Errorf("Must specify --image when not using --copy-to.")
 		}
 	}
 
 	// Image
-	if len(o.Image) == 0 {
-		return fmt.Errorf("--image is required")
-	}
-	if !reference.ReferenceRegexp.MatchString(o.Image) {
+	if len(o.Image) > 0 && !reference.ReferenceRegexp.MatchString(o.Image) {
 		return fmt.Errorf("Invalid image name %q: %v", o.Image, reference.ErrReferenceInvalidFormat)
```

Comment:
leftover from last time, but same rule applies, errors should start with lower case.